x-pack/plugins/observability/public/pages/cases/components/cases.tsx

```diff
@@ -45,14 +45,14 @@ export function Cases({ permissions }: CasesProps) {
   return (
     <>
       <CasesList
-        basePath={casesPath}
+        basePath={CASES_PATH}
         features={{ alerts: { sync: false, isExperimental: false } }}
         owner={[observabilityFeatureId]}
         permissions={permissions}
         ruleDetailsNavigation={{
-          href: (ruleId) => prepend(paths.observability.ruleDetails(ruleId)),
+          href: (ruleId) => prepend(paths.observability.ruleDetails(ruleId || '')),
```

Comment:
What will happen in case of not having a ruleId? Will we show an error on the page?

Reply:
Because previously, in `/config/paths.ts`, we had:

```
   ...
   ruleDetails: (ruleId?: string | null) =>
      ruleId ? `${RULES_PAGE_LINK}/${encodeURI(ruleId)}` : RULES_PAGE_LINK,
   ...
```

This has now made more strict as well as consistent with the other details pages (Alert, SLO): 

```
    ruleDetails: (ruleId: string) => `${OBSERVABILITY_BASE_PATH}${RULES_PATH}/${encodeURI(ruleId)}`,
```

Reply:
What will show in this case for rule details if there is no `ruleId`?